### PR TITLE
Bic extraction

### PR DESCRIPTION
**Pull request by SvenjaKernAi**
PR checklist:
- [x] Tested by creator on localhost:8000/docs
- [x] Tested by creator on refinery
- [x] Tested by reviewer on localhost:8000/docs
- [x] Tested by reviewer on refinery
- [x] (If added) common code tested in notebook/ script
- [x] Conforms with the agreed upon code pattern

### CODE REVIEW

File: bic.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    else:
+        printStuff("Invalid bic")
+
+
+'''
+testing the code
+'''
+bics = pd.read_excel("C:\Users\svenj\Downloads\blz-aktuell-xlsx-data.xlsx")
+
+
```
Comment (LeonardPuettmannKern) [inline bic.py:28]: @SvenjaKernAi Could you remove this file please? :)

Comment (SvenjaKernAi) [reply bic.py:28]: Deleted :)

**Comment by SvenjaKernAi** [comment]
The module is updated to the `__init__.py` in the extration-folder. 
Also, `bic.py` is deleted.

**Comment by SvenjaKernAi** [comment]
I changes the regex and push it on git.
Now the bic has to be at the begining on the string or has to have a whitespace before the bic itself (but no other characters). 
Additionally, after the requested 8 or 11 characters of the bic there has to be a whitespace, a special-character or the end of the string. So thats filter out all upper-cased words with more or less characters.

Nevertheless, there is a small gap, where a upper-cased word with exactly 8 or 11 character can be indentified as bic. My only idea to prevent that, is to make is list with all possible country-markers and to check whether one of them is used in space 5 and 6 (https://de.wikipedia.org/wiki/ISO-3166-1-Kodierliste). But this will exclude all country-name-changes that might happen.

**Comment by JWittmeyer** [comment]
I think we can't prevent false positives for the regex since it's only used to validate a bic not find it and the pattern ISO standard allows a lot. I don't see this as a big issue since we specifically mention that heuristics aren't meant to be exact. To better understand the issue from a user perspective we might want to include a false positive case in the examples.

Other sites seem to check the country code iso list but this can still result in false positives since the rest is unsure :D 
image
image
image

File: extractors/numbers/bic_extraction/code_snippet_refinery.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+```python
+import re
+
+ATTRIBUTE: str = "text" # only text attributes
+LABEL: str = "bic"
+
+def bic_extraction(record):
+    regex = re.compile(r'\b[A-Z0-9]{4,4}[A-Z]{2,2}[A-Z2-9][A-NP-Z0-9]([X]{3,3}|[A-WY-Z0-9]{1,1}[A-Z0-9]{2,2}|\s|\W|$)')
```
[the remark this answers is not in the document]
Comment (SvenjaKernAi) [reply extractors/numbers/bic_extraction/code_snippet_refinery.md:8]: Thanks, I changed the first 4 to letters

**Review by LeonardPuettmannKern — APPROVED** [review]
Good job. 👍

**Pull request merged**